plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -3496,6 +3496,7 @@ protected Answer execute(GetVmStatsCommand cmd) {
     }
 
     protected Answer execute(GetVmDiskStatsCommand cmd) {
+        // TODO: implement for Vmware
```

Comment:
Use/extend perfcounter, and see https://www.vmware.com/support/developer/converter-sdk/conv61_apireference/virtual_disk_counters.html for disk read/write and iops